scripts/docs/snippetadapter.js

```diff
@@ -80,12 +80,13 @@ module.exports = function snippetAdapter( snippets, options, umbertoHelpers ) {
 		snippets.add( snippetData );
 	}
 
-	// Remove snippets that do not match to patterns specified in `options.whitelistedSnippets`.
-	if ( options.whitelistedSnippets ) {
-		filterWhitelistedSnippets( snippets, options.whitelistedSnippets );
+	// Remove snippets that do not match to patterns specified in `options.allowedSnippets`.
+	if ( options.allowedSnippets ) {
+		filterAllowedSnippets( snippets, options.allowedSnippets );
 	}
 
-	console.log( `Building ${ snippets.size } snippets...` );
+	console.log( `Found ${ snippets.size } "{@snippet ...}" tags...` );
```

Comment:
I'd move this one to the loop above to log it only when specifying `--snippets` as it does not make much sense in a use without this option. Also "matching {@snippet} tags" would be better. Too many ellipsis and without "matching" it actually does not say the trurth as more tags were found.

Reply:
Moving the log before the `for..of` loop (as I understood) produces invalid results:

```
Found 132 matching {@snippet} tags.
[ 'classic-editor', 'build-classic-source' ] // --snippets
```

Filtering is done after the loop and that's why the log is here.

```
Found 8 matching {@snippet} tags.
[ 'classic-editor', 'build-classic-source' ] // --snippets
```